python/common.py

```diff
@@ -132,9 +129,17 @@ def __init__(self):
                           help="API endpoint, secure channel will be used if port ends with 443 (443, 8443, etc). "
                           "Default will use stt.tinkoff.ru:443 for recognition and tts.tinkoff.ru:443 for "
                           "synthesis.")
-        self.add_argument("--api_key", type=str, default=self._default_api_key, help="API key for JWT authentication.")
-        self.add_argument("--secret_key", type=str, default=self._default_secret_key, help="Secret key for "
-                          "HMAC-based JWT authentication.")
+        if self._default_api_key is None:
```

Comment:
We could also use [ConfigArgParse](https://pypi.org/project/ConfigArgParse/) if we dropped support of deprecated env variables.